.github/workflows/go.yml

```diff
@@ -146,11 +146,35 @@ jobs:
         key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
 
     - run: make deps
+
     - id: build
       run: make package-builder
+
     - name: package
+      id: run-package-builder
+      if: ${{ ! contains(matrix.os, 'macos') }}
       run: ${{ steps.build.outputs.binary }} make -debug --hostname=localhost --enroll_secret=secret
 
+    - name: package macOS
+      id: run-package-builder-macos
+      if: ${{ contains(matrix.os, 'macos') }}
+      shell: bash
+      run: |
+        # Capture last line of output, `Built packages in /var/some/location`
+        OUTPUT=$(${{ steps.build.outputs.binary }} make -debug --hostname=localhost --enroll_secret=secret --launcher_version=nightly | tail -n 1)
+        OUTPUT_PREFIX="Built packages in "
+        echo "launcher-pkg-path=${OUTPUT#"$OUTPUT_PREFIX"}" >> "$GITHUB_OUTPUT"
```

Comment:
Hrm. I can live with this, but it feels roundabout. `package-builder` supports a `--output_dir` option if you want to specify where it lands.

Reply:
Ok, better to set `--output_dir` to the current directory and use that in `Test install macOS` below?

Reply:
Updated! This is significantly more streamlined now -- thanks for the suggestion